pkg/replication/replication.go

```diff
@@ -257,9 +315,36 @@ func (r *replication) Enact() {
 	}
 	sort.Sort(order)
 
-	nodeQueue := make(chan types.NodeName)
+	nodeQueue := r.nodeQueue
+	if nodeQueue == nil {
```

Comment:
How would it come to pass that nodeQueue is not set? Is this the code that implements the "old" behavior?

Reply:
yeah, it will be unset if you use `InitializeReplication()` rather than `InitializeDaemonSetReplication`. I can make this more explicit here with a `r.UseNodeQueue()` or something like that